src/json_schema_factory.coffee

```diff
@@ -35,11 +36,33 @@ buildDefinitionFromConfig = (config, propertyIsRequired) ->
     when config.type is 'boolean'
       -> faker.random.array_element [true, false]
 
-    when config.format is 'date-time'
-      -> faker.date.between(new Date('2013-01-01'), new Date('2014-01-01'))
-
     when config.type is 'string'
-      -> faker.lorem.words().join ' '
+      switch config.format
+        when undefined
+          -> faker.lorem.words().join ' '
+
+        # see https://github.com/goodeggs/goodeggs-json-schema-validator for supported formats
+        when 'objectid'
```

Comment:
we just rewrote git history of goodeggs-json-schema-validator and made it public, so that is no longer an issue. And yes, objectid is still mongodb specific. However, since It is just one added format and it does not break for normal usage (without objectId), I would be inclined to leave it in.